eventually/team/migrations/0001_initial.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Merge migrations file to one.
0001_initial.py
0002_auto_20171029_2302.py
0003_auto_20171030_1150.py
0004_auto_20171030_1155.py
0005_auto_20171030_1314.py

output: only 0001_initial.py file.

Reply:
done